src/System.Diagnostics.Debug.SymbolReader/src/System/Diagnostics/Debug/SymbolReader/SymbolReader.cs

```diff
@@ -0,0 +1,120 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Reflection.Metadata;
+using System.Reflection.Metadata.Ecma335;
+using System.Reflection.PortableExecutable;
+
+namespace System.Diagnostics.Debug.SymbolReader
+{
+
+    public class SymbolReader
+    {
+    
+        /// <summary>
+        /// Checks availability of debugging information for given assembly.
+        /// </summary>
+        /// <param name="assemblyName">file name of the assembly</param>
+        /// <returns>true if debugging information is available</returns>
+        public static bool LoadSymbolsForModule(string assemblyName)
+        {
+            MetadataReader peReader, pdbReader;
+            
+            return GetReaders(assemblyName, out peReader, out pdbReader);
+        }
+    
+        /// <summary>
+        /// Returns method token and IL offset for given source line number.
+        /// </summary>
+        /// <param name="assemblyName">file name of the assembly</param>
+        /// <param name="fileName">source file name</param>
+        /// <param name="lineNumber">source line number</param>
+        /// <param name="methToken">method token return</param>
+        /// <param name="ilOffset">IL offset return</param>
+        public static void ResolveSequencePoint(string assemblyName, string fileName, int lineNumber, out int methToken, out int ilOffset)
```

Comment:
`assemblyName` is a file path, right? Maybe the parameter should be named `assemblyFileName` instead of `assemblyName` so it's more clear it is a path to a file and to avoid any confusion with being an [AssemblyName](https://msdn.microsoft.com/en-us/library/b6ccxba0(v=vs.110).aspx) (i.e. assembly FullName). Applies throughout. `assemblyFileName` is the same naming used in [`AssemblyBuilder.Save`](https://msdn.microsoft.com/en-us/library/8zwdfdeh(v=vs.110).aspx). Another alternative would be `assemblyFile`, which is used by [`AssemblyName.GetAssemblyName`](https://msdn.microsoft.com/en-us/library/system.reflection.assemblyname.getassemblyname(v=vs.110).aspx), but that wouldn't be consistent with the naming of the next parameter here, `fileName`.